src/CardsAPI/Hand.py

```diff
@@ -1,13 +1,11 @@
-from .Card import Card
+from src.CardsAPI.Card import Card
 from typing import List, Tuple
 
 
 class Hand(object):
-    def __init__(self, card_list: List[Card]=None, is_dealer_hand: bool=False,
-                 isSplit: bool = False):
-        if card_list is None:
-            card_list = []
-        self.card_list = card_list
+    def __init__(self, card_list: List[Card] = None, is_dealer_hand:
+                 bool = False, isSplit: bool = False):
```

Comment:
I don't like the fact that you initialize `is_dealer_had` and its typing not on the same line, it's not possible to move `is_dealer_hand` one line below ? To have parameter and its type near !?

Reply:
Absolutely, I'll fix it

Reply:
I'll wait your fix to accept my review :)